src/app/joined-students/joined-students.component.scss

```diff
@@ -0,0 +1,49 @@
+@import "../../variables";
+
+.list-container {
+  background-color: mat-color($accent);
+  // Fill the rest of the screen. (The parts not covered by the toolbar.)
+  flex: 1;
+  // Crop the contents if they're too big.
+  overflow: hidden;
+  width: 100%;
+  height: 100%;
+}
+
+.list-header {
+  display: flex;
+  flex-direction: row;
+  align-items: baseline;
+  justify-content: space-between;
+  margin: 2.5% 1.5%;
+}
+
+.join-code {
+  font-size: 3em;
+  font-family: 'Press Start 2P', cursive;
```

Comment:
You may have this font locally on your system but unless we include it in the app this defaults to comic  sans for most people. We should stick with the default font for now (Roboto)